### DISCUSSION TITLE
Introduce API Hook for Marketing Campaign

### DISCUSSION THREAD

**Discussion by Crazycolbster** [OpenRCT2/OpenRCT2#20681] [Feature requests]
This is a niche request, but it would be useful for my plugin to know whenever a player starts a marketing campaign, especially if I could know what type of campaign and what ride it applies to. If somebody could add that, I’d really appreciate it!

**Comment by Broxzier** [comment]
I can think of two hooks - one right before it starts and one right after. The first one could allow you to make changes before it actually starts. The second one is more like an event or notification to inform your plug-in.
